pkgs/applications/misc/electron-cash/default.nix

```diff
@@ -51,18 +52,20 @@ python3Packages.buildPythonApplication rec {
   doCheck = false;
 
   postInstall = ''
-    # Despite setting usr_share above, these files are installed under
-    # $out/nix ...
-    mv $out/${python.sitePackages}/nix/store"/"*/share $out
-    rm -rf $out/${python.sitePackages}/nix
+    # These files are installed under $out/homeless-shelter ...
+    mv $out/${python.sitePackages}/homeless-shelter/.local/share $out
+    rm -rf $out/${python.sitePackages}/homeless-shelter
 
     substituteInPlace $out/share/applications/electron-cash.desktop \
       --replace "Exec=electron-cash %u" "Exec=$out/bin/electron-cash %u"
+
+    wrapProgram $out/bin/electron-cash \
+      --prefix QT_PLUGIN_PATH : ${qtbase}/lib/qt-5.${lib.versions.minor qtbase.version}/plugins
```

Comment:
While this may make the applications work, this is something that will need to be cleaned up after an actual fix is built into nixpkgs, such as #44047.

Reply:
Yes it was meant to be a temporary fix